src/transformers/models/auto/tokenization_auto.py

```diff
@@ -323,6 +332,105 @@ def tokenizer_class_from_name(class_name: str):
             return c
 
 
+def get_tokenizer_config(
```

Comment:
If we ever want to make this public, put all the proper type annotations and docstring.